README.md

```diff
@@ -12,6 +12,9 @@ This is a Go client library for Ably.
 ~ $ go get -u github.com/ably/ably-go/ably
 ```
 
+## Supported Versions of Go
```

Comment:
For consistency with other headers in this file, could you please add vertical whitespace below this line. I recognise it doesn't affect the rendering in GitHub's UI, however it does affect those who wish to read it raw as well as those maintaining the file.